cartography/data/jobs/cleanup/crxcavator_import_cleanup.json

```diff
@@ -4,11 +4,6 @@
     "iterative": true,
     "iterationsize": 100
   },
-  {
-    "query": "MATCH (n:GSuiteUser) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
might want to keep this since you do a MERGE - If Gsuite module runs first, there's small chance this would cause issues with extra writes. Also, assume GSuite intel may not be running also for various reasons